Lib/pydoc.py

```diff
@@ -1682,6 +1682,13 @@ def describe(thing):
         return 'function ' + thing.__name__
     if inspect.ismethod(thing):
         return 'method ' + thing.__name__
+    if inspect.ismethodwrapper(thing):
+        return 'method wrapper ' + thing.__name__
+    if inspect.ismethoddescriptor(thing):
+        try:
+            return 'method descriptor ' + thing.__name__
```

Comment:
Although for `_lru_cache_wrapper` (a class in C, a function in Python) we would have 'method descriptor', I don't know if you want to special case this one (maybe with an exclusion list where you would add the types of 'thing' that shouldn't have "method descriptor").

Reply:
This is technically correct, it is a method descriptor.

Reply:
Yes, but I feel like it doesn't "help" you when you read the title (at least it could be confusing but changing it would be lying to the user =/)